lib/rubygems_plugin.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'rdoc/rubygems_hook'
+
+# To install dependency libraries of RDoc, you need to run bundle install.
+# At that time, rdoc/markdown is not generated.
+# If generate and remove are executed at that time, an error will occur.
+# So, we can't register generate and remove to Gem at that time.
+begin
+  require_relative 'rdoc/markdown'
```

Comment:
Right.
We can use one of `rdoc/markdown`, `rdoc/rd/block_parser` or `rdoc/rd/inline_parser` here.
This just wants to fix a bootstrap problem.

We want to run `bundle install` when we start developing cloned ruby/rdoc.
In the time, this `lib/rubygems_plugin.rb` is used. But RDoc isn't ready yet because `rdoc/markdown.rb` and so on aren't generated yet. So `bundle install` is failed.

This disables this `lib/rubygems_plugin.rb` only when the initial `bundle install` to fix the bootstrap problem.